services/modules/helm-charts/charts/guardian-guardian-service/templates/deployment.yaml

```diff
@@ -32,11 +32,54 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
-          resources:
-            {{- toYaml .Values.resources | nindent 12 }}
+          command: ["/bin/sh"]
+          args: ["entrypoint.sh"]
+          livenessProbe:
+            {{- toYaml .Values.livenessProbe | nindent 12 }}
+          readinessProbe:
+            {{- toYaml .Values.readinessProbe | nindent 12 }}
+          resources: 
+            limits:
+              cpu: "{{ .Values.resources.cpu}}"
+              memory: "{{ .Values.resources.memory}}"
+            requests:
+              cpu: "{{ .Values.resources.cpu}}"
+              memory: "{{ .Values.resources.memory}}"
           envFrom:
             - configMapRef:
                 name: {{ include "guardian-guardian-service.fullname" . }}-{{ .Release.Revision }}
+            - secretRef:
+                name: {{ include "guardian-guardian-service.fullname" . }}-{{ .Release.Revision }}
+            - configMapRef:
+                name: guardian-extensions-newrelic
```

Comment:
Again - should this be dynamic?

Reply:
will try to see if there any way to inject that, it bad because they are not in the same release